lawnchair/src/app/lawnchair/search/LawnchairAppSearchAlgorithm.kt

```diff
@@ -29,19 +32,29 @@ import kotlinx.coroutines.Dispatchers
 import me.xdrop.fuzzywuzzy.FuzzySearch
 import me.xdrop.fuzzywuzzy.algorithms.WeightedRatio
 import java.util.*
+import java.util.stream.Collectors
 
 class LawnchairAppSearchAlgorithm(context: Context) : LawnchairSearchAlgorithm(context) {
 
     private val appState = LauncherAppState.getInstance(context)
     private val resultHandler = Handler(Executors.MAIN_EXECUTOR.looper)
     private var enableFuzzySearch = false
+    private var hiddenApps: Set<String> = setOf()
```

Comment:
Let it be `val`

Reply:
couldn't make it val as it will get updated in init method , used lateinit insted

Reply:
Rechecked, it inited once, that's fine.